articles/how-to-insert-3d-objects-into-a-webpage-using-html-and-css/index.md

```diff
@@ -0,0 +1,320 @@
+}
+
+#icons #iconsLogo {
+    height: 300px;
+    width: 50px;
+    display: flex;
+    flex-direction: column;
+}
+
+#icons #iconsLogo i {
+    height: 50px;
+    width: 50px;
+    font-size: 30px;
+    margin: 10px;
+    color: grey;
+    display: grid;
+    place-items: center;
+    cursor: pointer;
+}
+```
+
+The results are as follows:
+![Page structure](/engineering-education/How-to-insert-3D-objects-into-a-webpage-using-HTML-and-CSS/structure-design.png)
+
+## Insert the 3D Object
+Let's now add the 3D model to the webpage. First, if the 3D model files are in a zip format, extract the downloaded model zip into files such as the ".bin" and ".gltf".
+
+> Data from the ".bin" file is used for buffering in buffers, which is then used by BufferViews, accessors, and mesh primitives
+
+Copy the files into the assets folder in the main directory. Make sure to check and see if the zip had additional files or images so that when copying the files, the file names may not conflict with those of the images in the assets folder. If they do conflict, rename the images in the assets folder first before pasting.
+In the div element with the "aside" id, let's import the 3D object and some of its attributes. For this case, we are going to use the type, source(src), alternative text (in case of any loading errors), auto-rotate(for a continuous rotation of the object), camera-controls(to allow one to view the object at their preferred angle of view), ar(to support Augmented Reality, AR, devices), and ios-src(to be used on supported iOS 12+ devices). Write the code below to import the 3D model in the webpage:
+ 
+ ```html
+<model-viewer src="assets/HTC_Vive_Headset.gltf" alt="VR Headset" auto-rotate camera-controls ar ios-src="assets/HTC_Vive_Headset.gltf">
+</model-viewer>
+ ```
+
+You can obtain the code above and even more attributes in [Model-viewer](https://modelviewer.dev/) site. Here you can also see some model attributes available for the object and their usages in the docs section. 
+
+In the live preview, it will output the following:
+![Inserting a 3D Model](/engineering-education/How-to-insert-3D-objects-into-a-webpage-using-HTML-and-CSS/Inserting-a-3D-model.png)
+
+The next step is to modify its appearance using CSS. Let's do this by adding the code below to our CSS file:
+
+```css
+#aSide {
+    height: 600px;
+    width: 600px;
+    position: absolute;
+    top: calc(50% - 250px);
+    right: 7%;
+}
+
+#aSide model-viewer {
+    height: 600px;
+    width: 600px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    border: none;
+}
+```
+
+**Final results:**
+![Final results](/engineering-education/How-to-insert-3D-objects-into-a-webpage-using-HTML-and-CSS/full-design.png)
+
+If you have reached here, this means that your 3D model is well inserted in the webpage, rotates at a uniform rate and the camera angle can be modified. 
+Well done! You have successfully inserted a 3D object into your webpage.
+
+This fixes #1645
```

Comment:
Remove this.

This must be added in the PR check list.

Reply:
Understood

Reply:
Done